assignments/boolean-birds/code.js

```diff
@@ -16,3 +16,23 @@
  *   - Globby Birds eat fish, mice, and nuts.
  *
  */
+const isFlobbyBird= (red,spotted) => red&&spotted
```

Comment:
These look good. As a matter of style you should put spaces around operators.

```suggestion
const isFlobbyBird = (red,spotted) => red && spotted;
```

It may seem a small thing but if code is consistently formatted it is much easier to spot problems. (As a hint, in the editor you can right click to pull up a menu that has an item "Format document" which will fix a lot of problems like this.